docs/docs/feature/keymaps.md

```diff
@@ -17,9 +17,23 @@ in the future by allowing user configs to add to the CMake application built by
 ## Big Picture
 
 All keyboard definitions (complete boards or shields) include the _default_ keymap for that keyboard,
-so ZMK can produce a "stock" firmware for that keyboard without any further modifications. For users
-looking to customize their keyboard's behavior, they can copy the stock `.keymap` file into their
-user config directory, and customize the keymap to their liking.
+so ZMK can produce a "stock" firmware for that keyboard without any further modifications. 
+
+For users looking to customize their keyboard's behavior, they can copy the stock `.keymap` file into their
+local [ZMK config directory](docs/feature/keymaps.md#ZMK-Configuration-Directories), and customize the keymap to their liking.
+
+## ZMK Configuration Directories
```

Comment:
I think this should likely end up in a dedicated new document that covers this in detail.

Reply:
Required documentation mostly covered in #135